app/components/agendapunt-overview-item.hbs

```diff
@@ -2,16 +2,14 @@
   <div class="au-c-card__header">
     <div class="au-o-grid au-o-grid--small">
       <div class="au-o-grid__item au-u-3-4@medium">
-        <AuHeading @level="2" @skin="4" property="dct:title">{{clean-spaces @agendapunt.titel}}</AuHeading>
+        <h2 class="au-c-heading--4" property="dct:title">{{clean-spaces @agendapunt.titel}}</</h2>
```

Comment:
I've removed the appuniversum components here because they add whitespace and for rdfa purposes we don't want that

Reply:
So ideally this is fixed in appuniversum itself? The problem is that the RDFa scraper adds the whitespace to the scraped data or something?